Formula/u/uffizzi.rb

```diff
@@ -66,8 +66,8 @@ class Uffizzi < Formula
   end
 
   resource "thor" do
-    url "https://rubygems.org/gems/thor-1.3.0.gem"
-    sha256 "1adc7f9e5b3655a68c71393fee8bd0ad088d14ee8e83a0b73726f23cbb3ca7c3"
+    url "https://rubygems.org/gems/thor-1.2.1.gem"
```

Comment:
this seems a bit off, 

https://github.com/UffizziCloud/uffizzi_cli/blob/develop/Gemfile.lock#L105